code/web/src/modules/user/api/action.test.js

```diff
@@ -0,0 +1,59 @@
+import * as actions from './actions.js'
+import axios from 'axios'
+import mockAxios from 'jest-mock-axios';
+jest.mock('axios');
+
+describe('Actions', () => {
+  let user, token;
+  beforeEach(() => {
+    user = {
+      details: {
+        name: "The User",
+        email: "user@crate.com",
+        role: "USER"
+      },
+      error: null,
+      isLoading: false,
+      isAuthenticated: true
+    }
+
+    token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MiwibmFtZSI6IlRoZSBVc2VyIiwiZW1haWwiOiJ1c2VyQGNyYXRlLmNvbSIsInJvbGUiOiJVU0VSIiwiaWF0IjoxNTk4MzMzMTQwfQ.EodZNLw5drqn7FzMTTt4EHujMoAkOeqJaJ5qZXidjho"
+  })
```

Comment:
I'm curious where this token came from. Was it in a response or is it just mock data?

Reply:
This is the token that gets generated for the authorization I believe. 
I noticed it on the log in and had to add it for the test fn to run properly.

Reply:
That makes sense. We've had trouble in the past using actual tokens in tests because sometimes they expire, but it sounds like that isn't an issue here.